### PR TITLE
Updated custom-vision-model-v2 to custom-vision-model

### PR DESCRIPTION
**Pull request by sushilkm**
This commit is aimed to remove the references of custom-vision-model-v2.
All references would be directed to one pipeline name model custom-vision-model.

Fixes [issues/85](https://github.com/Azure/Azure-Orbital-Analytics-Samples/issues/85)

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -389,7 +389,7 @@ To run the pipeline, open the Synapse Studio for the Synapse workspace that you
     > SELECT *, ST_Distance(ST_SetSRID(ST_MakePoint(given_long, given_lat), 4326), location) AS dist FROM aioutputmodelschema.cvmodel
   - More documentation on queries can be found [here](https://postgis.net/workshops/postgis-intro/geography.html)
 
-# Running the pipeline (Custom Vision Model V2)
+# Running the pipeline (Custom Vision Model)
```
Comment (karthick-rn) [inline deploy/README.md:392]: The V2 section has been renamed. Was the older section removed?

File: deploy/README.md [hunk 2]
```diff
@@ -190,7 +190,7 @@ or
 Arguments | Required | Type | Sample value
 ----------|-----------|-------|---------------
 environmentCode | yes | string | aoi
-pipelineName | yes | string | Allowed Values: custom-vision-model, custom-vision-model-v2
+pipelineName | no | string | Allowed Value: custom-vision-model
```
Comment (karthick-rn) [inline deploy/README.md:193]: This should be `yes` as well

**Comment by karthick-rn** [comment]
Overall, the changes looks good to me. 

Few questions:

- Was the solution tested manually after these changes? 
- Noticed the PR has been submitted as a branch in-line with main, usually we do it via fork against main. Can you check?
-  Will these changes be applied to the other active branch as well?

File: deploy/gallery/instructions.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 If you've landed on this page from Synapse gallery then Congrats! -- you are in the right place.
 
-The instructions on this page will guide you through configuring the pipeline template to successfully run a custom vision model for object detection (for example, detecting swimming pool objects). You can also use the pipeline template to explore other object detection use cases as well. Currently, the template only supports custom vision model v2. 
+The instructions on this page will guide you through configuring the pipeline template to successfully run a custom vision model for object detection (for example, detecting swimming pool objects). You can also use the pipeline template to explore other object detection use cases as well. Currently, the template only supports custom vision model.
```
[the remark this answers is not in the document]
Comment (sushilkm) [reply deploy/gallery/instructions.md]: Done

File: deploy/README.md (hunk 1 above)
Comment (sushilkm) [reply deploy/README.md]: Removed, thanks for catching!

File: deploy/README.md (hunk 2 above)
Comment (sushilkm) [reply deploy/README.md:193]: package.sh has a default value. so it is not a required parameter

File: deploy/README.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Arguments | Required | Type | Sample value
 ----------|-----------|-------|-------
 environmentCode | yes | string | aoi
 location | yes | string | westus
-pipelineName | no | string | Allowed values: custom-vision-model, custom-vision-model-v2
+pipelineName | no | string | Allowed value: custom-vision-model
```
[the remark this answers is not in the document]
Comment (sushilkm) [reply deploy/README.md:64]: setup.sh has a default value. so it is not a required parameter

[the remark this answers is not in the document]
Comment (karthick-rn) [reply deploy/README.md:64]: I don't see the `setup.sh` or `package.sh` scripts having a default value. I think that may be the case in the CI/CD.

[the remark this answers is not in the document]
Comment (sushilkm) [reply deploy/README.md:64]: Following lines show the default value for `setup.sh` and `package.sh`

https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/97eb4cd0d55ca8daa67358c0cd85d900a62959e5/deploy/setup.sh#L8 for setup.sh
https://github.com/Azure/Azure-Orbital-Analytics-Samples/blob/97eb4cd0d55ca8daa67358c0cd85d900a62959e5/deploy/package.sh#L9 for package.sh

File: deploy/README.md (hunk 2 above)
Comment (karthick-rn) [reply deploy/README.md:193]: I was comparing the code that is in the main branch but didn't recognise that the default value was added on this PR.

**Review by karthick-rn — APPROVED** [review]

File: deploy/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (karthick-rn) [reply deploy/README.md:64]: I was comparing the code that is in the main branch but didn't recognise that the default value was added on this PR.

**Pull request merged**